video/coktel_decoder.cpp

```diff
@@ -1543,6 +1543,93 @@ Graphics::PixelFormat IMDDecoder::getPixelFormat() const {
 	return Graphics::PixelFormat::createFormatCLUT8();
 }
 
+class DPCMStream : public Audio::AudioStream {
+public:
+	DPCMStream(Common::SeekableReadStream *stream, int rate, int channels, bool oldStereo) {
+		_stream = stream;
```

Comment:
could also:
```
assert(channels <= 2);
```
As _buffer is hard-coded with a length of two and indexed up to the channel count.